purchases/src/main/kotlin/com/revenuecat/purchases/VirtualCurrencyInfo.kt

```diff
@@ -0,0 +1,40 @@
+package com.revenuecat.purchases
+
+import android.os.Parcelable
+import com.revenuecat.purchases.models.RawDataContainer
+import com.revenuecat.purchases.utils.JSONObjectParceler
+import kotlinx.parcelize.IgnoredOnParcel
+import kotlinx.parcelize.Parcelize
+import kotlinx.parcelize.TypeParceler
+import org.json.JSONObject
+
+@Parcelize
+@TypeParceler<JSONObject, JSONObjectParceler>()
+data class VirtualCurrencyInfo(
+    val balance: Long,
+    private val jsonObject: JSONObject,
```

Comment:
I'm not super familiar with the pattern here of including the `jsonObject` in the data class but saw it in several other model data classes. Is it appropriate to keep that here?